packages/forklift-console-plugin/src/modules/Overview/utils/helpers/OverviewStateContextProvider.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  createContext,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import { produce } from 'immer';
+
+export interface CreateOverviewStateContextData {
+  hideWelcomeCardByContext: boolean;
```

Comment:
```suggestion
  hideWelcomeCard: boolean;
```

i prefer shorter name, it's a context data, the "by context" is a known thing

Reply:
Actually here I prefer the longer naming version since when used, it's compared to the `hideWelcomeCard` mode as known by the entity before updated by the context.